packages/amplify_authenticator/lib/src/widgets/authenticator_banner.dart

```diff
@@ -21,60 +21,124 @@ import 'package:flutter/material.dart';
 MaterialBanner createMaterialBanner(
   BuildContext context, {
   required StatusType type,
-  required Widget content,
-  required List<Widget> actions,
+  required String message,
+  required void Function() actionCallback,
 }) {
-  final margin = MediaQuery.of(context).viewPadding.top;
+  final theme = Theme.of(context);
+
+  final colorsChoices = getColors(theme.bannerTheme.backgroundColor,
+      theme.bannerTheme.contentTextStyle?.color, type, theme);
 
   return MaterialBanner(
     key: keyAuthenticatorBanner,
-    leading: Icon(type.icon),
-    padding: EdgeInsetsDirectional.only(
-      start: 16.0,
-      top: 2.0 + margin,
-      bottom: 4.0,
+    backgroundColor: colorsChoices.background,
+    leading: Icon(type.icon, color: colorsChoices.foreground),
+    content: Center(
+      child: Row(
+        crossAxisAlignment: CrossAxisAlignment.center,
+        children: [
+          Expanded(
+            child: Text(
+              message.trimRight(),
```

Comment:
Doesn't need to be changed but I find this a little confusing over just `.trim()` or instead of trimming the text at the source. It doesn't really matter here, but just a general comment for the future.